minion/templates/namespace.yaml

```diff
@@ -3,5 +3,5 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: {{ .Release.Name }}
+  name: {{ .Release.Namespace }}
```

Comment:
What do you think about deleting this file altogether along with the createNamespace config option? The more proper way to do this with helm usually is to use `--create-namespace` if they want the namespace created and I don't think we need to maintain backward compatibility with the minion chart.